static/src/javascripts/projects/commercial/modules/dfp/breakpoint-name-to-attribute.ts

```diff
@@ -0,0 +1,4 @@
+const breakpointNameToAttribute = (breakpointName: string): string =>
+	breakpointName.replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase();
```

Comment:
I think extracting the RegExp into a constant will prevent it from being generated each time `breakpointNameToAttribute` is called.

See https://github.com/guardian/libs/pull/162#discussion_r623840854